spec/acceptance/ntp_parameters_spec.rb

```diff
@@ -73,14 +73,14 @@
   describe 'config_template' do
     before :all do
       modulepath = default['distmoduledir']
-      shell("mkdir -p #{modulepath}/test/templates")
+      on(default, "mkdir -p #{modulepath}/test/templates")
       # Add spurious template logic to verify the use of the correct template rendering engine
-      shell("echo '<% [1].each do |i| %>erbserver<%= i %><%end %>' >> #{modulepath}/test/templates/ntp.conf.erb")
+      on(default, "echo '<% [1].each do |i| %>erbserver<%= i %><%end %>' >> #{modulepath}/test/templates/ntp.conf.erb")
```

Comment:
These need to use `shell_ex` (see https://github.com/puppetlabs/beaker-testmode_switcher#reference) otherwise local mode won't work.